lib/README.txt

```diff
@@ -0,0 +1,45 @@
+This is a placeholder to put Hadoop native libraries -it's a component
```

Comment:
This file is duplicated from lib/native/README.txt - is there a reason?

Reply:
Not any. Inadvertently added